fdbserver/SkipList.cpp

```diff
@@ -795,99 +796,6 @@ class MiniConflictSet2 : NonCopyable {
 	}
 };
 
-class MiniConflictSet : NonCopyable {
```

Comment:
Given that you have already done a profile of a resolver, did your profiles show equal amounts of time being spent in here as in MiniConflictSet2?  If this is faster, I'd kind of prefer to keep it as the default implementaiton?  I'd hope that this implementation was benchmarked to be faster than MiniConflictSet2, and the ASSERT() below shows that they are equivalent for everything that this has run on so far...  but the bottom of this file also doesn't have a benchmark, so it's nontrivial to figure out by how much.